### ISSUE TITLE
[jasper/pytorch] nan loss for val with WER of 1.0

### ISSUE THREAD

**Issue by raotnameh** [NVIDIA/DeepLearningExamples#466]
while training it on our won in house dataset the val loss comes to nan and WER for val comes 1.0.
We use the standard configurations. Apart form that everything works fine for LibriSpeech dataset.

Additionally, if we use a model trained on LIbriSpeech dataset then it works for our in house dataset too.

**Comment by alancucki** [comment]
Hi @raotnameh ,
sorry for a late reply. Is the issue solved?

In a case like this, I'd first look for abnormalities in the data (erroneous transcripts, too short audio, etc.), and make sure, that the learning schedule is adjusted to the size of the new dataset.

**Comment by raotnameh** [comment]
@alancucki We removed the files which were giving NAN loss. After this everything works like a charm. Thanks for the reply.
